content/en/docs/ops/configuration/telemetry/telemetry-v1-vs-v2/index.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Differences between Telemetry v1 and v2
+description: Tradeoffs and gaps between Telemetry v1 and v2.
+weight: 10
+owner: istio/wg-policies-and-telemetry-maintainers
+test: no
+---
+
+Providing uniform telemetry and extension mechanisms to extend the functionality
+in Istio have been the driving factors for the architectural choices made in
+Istio. From early days, mixer aka Telemetry v1 provided both of these
+functionalities where an out-of-process service gathered attributes from the
+workload Envoy proxies, combined it with platform data like Kubernetes services
+and deployment associations and generated rich telemetry data like Prometheus
```

Comment:
```suggestion
and deployment associations, and generated rich telemetry data like Prometheus
```

I made some suggestions, but this sentence has a lot of information that may make it hard for people to understand. I'd recommend breaking it up into shorter, sequential sentences.